charts/pega/KafkaClusterRequirement.md

```diff
@@ -50,4 +50,54 @@ User:\<user-name\> |GROUP  | * | ALL   | ALLOW      | LITERAL
 User:\<user-name\> |CLUSTER  | \<Cluster-Name\> | IDEMPOTENT_WRITE   |ALLOW      | LITERAL
 
 
+### Stream (externalized Kafka service) test example with bitnami kafka
+
+For testing purpose you can install bitnami kafka using below commands 
```

Comment:
Can you tweak this before you merge so it's proper English??

For testing purpose, deploy a bitnami kafka cluster using the provided Helm command example where `my-kafka` is the namespace of your Kafka service deployment.

Reply:
you mean add the below sentence? is it now not approved? I cant see the changes yet.. please let me know if I should resubmit the PR

For testing purpose, deploy a bitnami kafka cluster using the provided Helm command example where my-kafka is the namespace of your Kafka service deployment.

Reply:
I think it is useful to add that sentence, but I'm not going to block your work. I'm just not going to let improper English slip by during a review, since we should uphold a standard. It's your choice, not mine. I approved the PR.

Reply:
Hi @taz-pega-work  please approve the changes